lldb/test/API/functionalities/multiple-slides/TestMultipleSlides.py

```diff
@@ -12,6 +12,10 @@
 class MultipleSlidesTestCase(TestBase):
     NO_DEBUG_INFO_TESTCASE = True
 
+    # The intermediate object main.o is compiled without debug info, but
+    # a.out is linked with `-gdwarf` on Windows. This creates a PDB.
```

Comment:
Why does -gdwarf make LLD create a PDB? Seems counterintuitive. Is that just some odd side effect?

Reply:
I think it's because of this:
https://github.com/llvm/llvm-project/blob/7b98280b6b7cb89b141a5874ff9ee3ce72dab92a/clang/lib/Driver/ToolChains/MSVC.cpp#L167-L168

Reply:
Opened #165614.